frontend/packages/ux-editor/src/containers/DesignView/PageAccordion/NavigationMenu/InputPopover/InputPopover.test.tsx

```diff
@@ -17,20 +17,39 @@ const mockSaveNewName = jest.fn();
 const mockOnClose = jest.fn();
 
 const defaultProps: InputPopoverProps = {
+  disabled: false,
   oldName: mockOldName,
   layoutOrder: mockLayoutOrder,
   saveNewName: mockSaveNewName,
   onClose: mockOnClose,
-  open: true,
-  trigger: <button>My trigger</button>,
 };
 
 describe('InputPopover', () => {
-  const user = userEvent.setup();
   afterEach(jest.clearAllMocks);
 
+  it('does hides dropdown menu item by default when not open', () => {
+    render(<InputPopover {...defaultProps} />);
+
+    const input = screen.queryByLabelText(textMock('ux_editor.input_popover_label'));
+    expect(input).not.toBeInTheDocument();
+  });
+
+  it('opens the popover when the dropdown menu item is clicked', async () => {
+    render(<InputPopover {...defaultProps} />);
+
+    const input = screen.queryByLabelText(textMock('ux_editor.input_popover_label'));
+    expect(input).not.toBeInTheDocument();
+
+    await openDropdownMenuItem();
+
+    const inputAfter = screen.getByLabelText(textMock('ux_editor.input_popover_label'));
+    expect(inputAfter).toBeInTheDocument();
+  });
+
   it('calls the "saveNewName" function when the confirm button is clicked', async () => {
+    const user = userEvent.setup();
```

Comment:
I like your idea, but the testing library says this in [their documentation](https://testing-library.com/docs/user-event/intro/):
“*We recommend invoking [userEvent.setup()](https://testing-library.com/docs/user-event/setup) before the component is rendered. This can be done in the test itself, or by using a setup function. We discourage rendering or using any userEvent functions outside of the test itself - e.g. in a before/after hook - for reasons described in ["Avoid Nesting When You're Testing"](https://kentcdodds.com/blog/avoid-nesting-when-youre-testing).*”

Reply:
Ok then It is better to keep them as they are. 😊